website/pages/docs/concepts/writing-styles.md

```diff
@@ -402,19 +406,19 @@ const styles = css({
 })
 ```
 
-The hash generated classnames will look like:
+The hash generated css will look like:
```

Comment:
Made this change because hash applies to both classnames and variables by default